LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
It is, so for now we can say that articles are all rights reserved, but I don't know how it would work with people contributing to blog posts.